pkg/machines/components/nicBody.jsx

```diff
@@ -0,0 +1,186 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import * as Select from "cockpit-components-select.jsx";
+import { getNetworkDevices } from '../helpers.js';
+import cockpit from 'cockpit';
+
+import './nic.css';
+
+const _ = cockpit.gettext;
+
+const NetworkModelRow = ({ idPrefix, onValueChanged, dialogValues, osTypeArch, osTypeMachine }) => {
+    const availableModelTypes = [
+        { name: 'virtio', desc: 'Linux, perf' },
+        { name: 'e1000e', desc: 'PCI' },
+        { name: 'e1000', desc: 'PCI, legacy' },
+        { name: 'rtl8139', desc: 'PCI, legacy' }];
+    const defaultModelType = dialogValues.networkModel;
+
+    if (osTypeArch == 'ppc64' && osTypeMachine == 'pseries') {
+        availableModelTypes.push('spapr-vlan');
```

Comment:
Hmm, the format of the available Model Types is { name: NAME, desc: DESC }  - here you just push a string.

Reply:
This is the stuff which I pulled from other component and did not changed. I have no idea, what description should go next to "spapr-vlan" since i don't know that model. 
So I changed a component to use the _desc_ value optionally.

Reply:
All comments resolved.